docs/datasources/flow_datasource.adoc

```diff
@@ -0,0 +1,63 @@
+= Flow Datasource
+
+== General
+
+The Flow Datasource can be used to retrieved application and conversation related metrics derived from flow data stored by OpenNMS.
+
+The query editor supports the following functions:
+
+[options="header, autowidth"]
+|===
+| Category  | Name                    | Description
+| Combine   | `topN`                  | Limit the results to the Top N conversations/applications when sorted by total bytes.
+| Combine   | `includeOther`          | Combine and include other conversations/applications that fall outside of the Top N.
+| Filter    | `withExporterNode`      | Filter for flows related to the given exporter node.
+| Filter    | `withIfIndex`           | Filter for flows related to the given interface index.
+| Transform | `perSecond`             | Convert the values to a per/second rate.
+| Transform | `negativeEgress`        | Make the egress values negative.
+| Transform | `negativeIngress`       | Make the ingress values negative.
+| Transform | `asTableSummary`        | Generate a tabular summary instead of time series data.
+| Transform | `combineIngressEgress`  | Sum ingress and egress values together.
+|===
+
+== Template Queries
+
+=== exporterNodesWithFlows
+
+The `exporterNodesWithFlows()` query will return a list of nodes which have exported flow data to OpenNMS.
```

Comment:
I guess the node must be known by OpenNMS, as exporting flows to OpenNMS is also possible if the exporter is not known by OpenNMS.

Reply:
I added a note bellow.